CONTRIBUTING.md

```diff
@@ -148,4 +148,4 @@ Contributing to `loopback-workspace` is easy. In a few simple steps:
 ```
 
 [Google C++ Style Guide]: https://google.github.io/styleguide/cppguide.html
-[Google Javascript Style Guide]: https://google.github.io/styleguide/javascriptguide.xml
\ No newline at end of file
+[Google Javascript Style Guide]: http://loopback.io/doc/en/contrib/style-guide.html
```

Comment:
Change the `Google JavaScript Style Guide` to `LoopBack style guide` and remove the c++ style guide since we don't use any C++ code here. Feel free to ignore the second comment though because we need to update `CONTRIBUTING.md` in whole anyways.